acceptance-tests/bosh_helpers.go

```diff
@@ -234,9 +253,9 @@ func boshInstances() []boshInstance {
 	return output.Tables[0].Rows
 }
 
-func deleteDeployment() {
+func deleteDeployment(boshDeployment string) {
 	By("Deleting the HAProxy deployment")
```

Comment:
can we print the `boshDeployment` string here? maybe helps in debugging later

Reply:
Fixed